InnerCircle.Authentication.Service/Services/InnerCircleHttpClient.cs

```diff
@@ -17,23 +17,25 @@ public InnerCircleHttpClient(IOptions<InnerCircleServiceUrls> urls)
 
         public async Task SendPasswordCreationLink(string corporateEmail, string passwordResetToken)
         {
-            var mailSenderLink = $"{_urls.MailServiceUrl}/mail/send-welcome-link";
+            var mailSenderLink = $"{_urls.MailServiceUrl}mail/send-welcome-link";
```

Comment:
return back the symbol "/". "_urls.MailServiceUrl" shouldn't finish with symbol "/"